### PR TITLE
Stardew valley: Game version documentation

### PR DESCRIPTION
**Pull request by agilbert1412**
## What is this fixing or adding?
With the 1.6 release of the game on Steam soon, the game needs to be downgraded on Steam to work for Archipelago. It will  be a huge source of tech support requests if not specified, so I added documentation about downgrading to the correct version, for 5.x.x
When 6.x.x releases, and downgrading is no longer required, this section will be removed
I also fixed the indent on some existing documentation in the game page

## How was this tested?
It's documentation

## If this makes graphical changes, please attach screenshots.
image

image

### CODE REVIEW

**Review by benny-dreamly — APPROVED** [review]
looks good

File: worlds/stardew_valley/docs/setup_en.md [hunk 1]
```diff
@@ -16,6 +16,13 @@
     * It is **not** recommended to further mod Stardew Valley with unsupported mods, although it is possible to do so. Mod interactions can be unpredictable, and no support will be offered for related bugs.
     * The more unsupported mods you have, and the bigger they are, the more likely things are to break.
 
+## Notes About Game Versions
+The randomizer is built for a specific version of Stardew Valley. When a new version gets released, we will try our best to support it as fast as possible, but can't make promises.
```
Comment (NewSoupVi) [inline worlds/stardew_valley/docs/setup_en.md:20]: I personally like when pronouns are repeated (but we can't make promises), but I'm not an expert, maybe that's bad writing or sth

**Review by NewSoupVi — APPROVED** [review]

File: worlds/stardew_valley/docs/setup_en.md (hunk 1 above)
Comment (benny-dreamly) [reply worlds/stardew_valley/docs/setup_en.md:20]: ooh yeah I uh missed that... that does look better if you do repeat the pronouns (Kaito is Quebecois)

Comment (agilbert1412) [reply worlds/stardew_valley/docs/setup_en.md:20]: Sure, that sounds better imo

**Review by Ixrec — APPROVED** [review]
I've never played Stardew Valley so can't comment on the game-specific aspects of this PR, but the wording of the "you may need to downgrade" section seems good to me. Specifically, I think it makes sense if you aren't already familiar with the issue, is very unlikely to become technically incorrect later, and is likely to help mitigate confusion if/when a major new version of the vanilla game comes out and Archipelago has to wait a release cycle to be updated for it.

**Review by Jouramie — APPROVED** [review]
Hopefully that will answer some future questions before they're asked in Discord. 🤞

File: worlds/stardew_valley/docs/setup_en.md [hunk 2]
```diff
@@ -16,6 +16,13 @@
     * It is **not** recommended to further mod Stardew Valley with unsupported mods, although it is possible to do so. Mod interactions can be unpredictable, and no support will be offered for related bugs.
     * The more unsupported mods you have, and the bigger they are, the more likely things are to break.
 
+## Notes About Game Versions
+The randomizer is built for a specific version of Stardew Valley. When a new version gets released, we will try our best to support it as fast as possible, but we can't make promises.
```
Comment (Exempt-Medic) [inline worlds/stardew_valley/docs/setup_en.md:20]: Usually the word "any" is used here, but it's up to you.
```suggestion
The randomizer is built for a specific version of Stardew Valley. When a new version gets released, we will try our best to support it as fast as possible, but we can't make any promises.
```

**Review by Exempt-Medic — APPROVED** [review]

**Review by Ixrec — APPROVED** [review]

**Review by Jouramie — APPROVED** [review]

**Review by ThePhar — APPROVED** [review]

**Pull request merged**